pom.xml

```diff
@@ -741,6 +741,13 @@
             </exclusions>
         </dependency>
 
+        <!-- used in blueocean-pipeline-editor test -->
```

Comment:
NIT: since this is just used in the test, should it be defined there?

Reply:
Looked to me like dependencies are all defined in `<dependencyManagement>` here even if they're only used in one submodule, but eh, I don't care. =)